File: src/main/java/com/javahere/reachyourgoal/dto/UserDTO.java
package com.javahere.reachyourgoal.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDTO {
    private String firstname;
    private String lastname;
    private String email;
    private String username;
    private String password;
}

File: src/main/kotlin/me/javahere/reachyourgoal/domain/dto/UserDto.kt
package me.javahere.reachyourgoal.domain.dto

import me.javahere.reachyourgoal.domain.id.UserId

class UserDto(
    val id: UserId,
    val firstName: String,
    val lastName: String,
    val username: String,
    val email: String,
    val isConfirmed: Boolean,
)
